### DISCUSSION TITLE
new release

### DISCUSSION THREAD

**Discussion by extrawurst** [maxcountryman/axum-login#82] [Q&A] [answered]
can we get a new release cut?

**Comment by extrawurst** [comment]
this would be very helpful for anyone currently having to pin to a git release branch to have the tower-update. this also means no one will be able to do a cargo release for the time being as they won't allow pinning to a git hash

**Comment by maxcountryman** [comment]
Sorry for the delay, yes I will cut a new release here.

There's various upstream things that are pending, but we can release ahead of that and then address when those land.

**Comment by maxcountryman** [comment, marked as answer]
This should be addressed with `v0.6.0`.
